src/modules/reporting/controllers.js

```diff
@@ -0,0 +1,5 @@
+const reportsConnector = require('../../lib/connectors/reporting/');
+
+const getReport = request => reportsConnector.getReport(request.params.reportIdentifier);
```

Comment:
As with UI layer, I think it would be more efficient to pipe this if the reporting module is responding with a CSV file